.github/workflows/ci.yaml

```diff
@@ -12,8 +12,8 @@ on:
 
 concurrency:
   # Cancel previous actions from the same PR: https://stackoverflow.com/a/72408109
```

Comment:
I iterated on the comment a bit. Seen here: https://github.com/aspect-build/rules_js/pull/1502/files

Reply:
need to update rules_jasmine too?

Reply:
rules_jasmine has the latest incarnation as well now https://github.com/aspect-build/rules_jasmine/blob/ef94648234b75a7726da71fc2dca5334096a9321/.github/workflows/ci.yaml#L14